### ISSUE TITLE
Chrono Trigger - Map Screen Continents Disappear

### ISSUE THREAD

**Issue by Awakened0** [SourMesen/Mesen-S#162]
When you press Select in the overworld in CT, a map screen appears:

bsnes:
![ct bsnes](https://user-images.githubusercontent.com/4565863/89005673-0dd1b100-d2d3-11ea-91a2-519e35bf03f2.png)

Mesen-S:
![ct Mesen-S](https://user-images.githubusercontent.com/4565863/89005688-13c79200-d2d3-11ea-9450-5cb22ad24d15.png)

Both the libretro core and the latest Windows dev build exhibit the same issue.

**Comment by Awakened0** [comment]
Also, occasionally I've had this happen to the background of the Epoch time period screen. I'm not sure what causes it; it seems random and clears up once I move to a different time period. [Here is a save file next to the Epoch to help reproduce.](https://github.com/SourMesen/Mesen-S/files/5038889/CT.Epoch.Save.zip)


![ct epoch bg](https://user-images.githubusercontent.com/4565863/89600012-fd6b8a00-d82e-11ea-9a98-9238e3f295ff.png)

**Comment by Max833** [comment]
Yeah, use the stable version for now. Sour fixed a bug (https://github.com/SourMesen/Mesen-S/issues/145) and this caused this issue.
